### ISSUE TITLE
Conda Install Package Not Found

### ISSUE THREAD

**Issue by ShvetankPrakash** [srivatsankrishnan/oss-arch-gym#24]
I am trying to run our step by step [here](https://oss-archgym.readthedocs.io/en/latest/installation.html) on a fresh VM instance in preparation for tutorial to make sure everything works properly based on our documentation but am getting the following issue:

```
(base) ~/oss-arch-gym$ conda env create -f environment.yml
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound: 
  - torch
  - qhoptim
 ```

This may just be an issue with this VM setup since it has worked previously but I am adding this issue to see if it is reproducible by others so we can resolve it if it is.

**Comment by srivatsankrishnan** [comment]
@nx6xe23 @shreyasgrampurohit I think these were changes that you guys must have added with the `Node` in the ML-proxy pipeline. Can you fix this please?

**Comment by nx6xe23** [comment]
I have created a new branch `environment-lib-fix`, where I have added `torch` and `qhoptim` in the `requirements.txt`. Although a new error stating that there are no versions of envlogger found,

<img width="798" alt="image" src="https://github.com/srivatsankrishnan/oss-arch-gym/assets/81481005/6ec8e50f-8056-4815-a76a-c2a7c24a7371">

Changing the python version to 3.10 does not help and changing to 3.11 creates a lot of conflicts.

**Comment by srivatsankrishnan** [comment]
I think this is the issue with MacOS. Envlogger is only supported in linux. Can you test this on a Linux box?
https://github.com/google-deepmind/envlogger#getting-started

**Comment by ShvetankPrakash** [comment]
I tried it on a Linux machine and also had this error today. Mariam also
had this error.

**Comment by ShvetankPrakash** [comment]
Can you submit the pull request @nx6xe23 for the updates?

Also @srivatsankrishnan the envlogger comes from which package?

**Comment by nx6xe23** [comment]
Created the pull request [#32](https://github.com/srivatsankrishnan/oss-arch-gym/pull/32).

**Comment by ShvetankPrakash** [comment]
Great thanks @nx6xe23 !

There is still one issue though @srivatsankrishnan . Envlogger requries GLIBC version 2.28 but it seems ubuntu package manager latest version is 2.27 (at least for me it is). I could install GLIBC2.28 from source but I don't think this is something we should ask of users. Mariam also ran into this issue. Not sure what you think we should do here @srivatsankrishnan ?

**Comment by ShvetankPrakash** [comment]
Okay I checked and GLIBC>2.27 is just not available on 18.04 but fine on Ubuntu 20.04 which I think we are saying is minimum version now so this is fine and closing this issue

**Comment by ShvetankPrakash** [comment]
PR #32 merged.
